src/tests

```diff
@@ -52,3 +52,5 @@ p({"class":"foam.nanos.test.Test","id":"FoldAndReduceTest","status":2,"code":"im
 p({"class":"foam.nanos.test.Test","id":"FoldAndReduceTest","passed":10,"failed":0,"lastRun":"2019-06-17T19:39:02.373Z","lastDuration":80,"status":1,"output":"SUCCESS: Stock DAO has expected number of items\n\nHourly data validation\nSUCCESS: Hourly DAO has expected number of items\nSUCCESS: [] fields are set incorrectly\nSUCCESS: [] fields are set incorrectly\nSUCCESS: [] fields are set incorrectly\n\nDaily data validation\nSUCCESS: Daily DAO has expected number of items\nSUCCESS: [] fields are set incorrectly\nSUCCESS: [] fields are set incorrectly\n\nWeekly data validation\nSUCCESS: weekly DAO has expected number of items\nSUCCESS: [] fields are set incorrectly\nDone\n"})
 p({"class":"foam.nanos.test.Test","id":"FoldAndReduceTest","status":2,"output":""})
 p({"class":"foam.nanos.test.Test","id":"FoldAndReduceTest","lastRun":"2019-06-17T19:39:12.053Z","lastDuration":57,"status":1,"output":"SUCCESS: Stock DAO has expected number of items\n\nHourly data validation\nSUCCESS: Hourly DAO has expected number of items\nSUCCESS: [] fields are set incorrectly\nSUCCESS: [] fields are set incorrectly\nSUCCESS: [] fields are set incorrectly\n\nDaily data validation\nSUCCESS: Daily DAO has expected number of items\nSUCCESS: [] fields are set incorrectly\nSUCCESS: [] fields are set incorrectly\n\nWeekly data validation\nSUCCESS: weekly DAO has expected number of items\nSUCCESS: [] fields are set incorrectly\nDone\n"})
+p({"class":"foam.nanos.test.Test","id":"Multi Line Output","status":2,"code":"file = java.io.File.createTempFile(\"temp\", null);\nfile.deleteOnExit();\noutputter = new foam.lib.json.Outputter(x, file);\nstring = \"\\\\\\t line1 \\n   \\\\n  \\n line2 \\n line3 \\\" \\\\ \";\nuser = new foam.nanos.auth.User(x);\nuser.setId(24);\nuser.setFirstName(string);\nuser.setLastName(\"Fjorenborenborkenberger\");\n\noutputter.setMultiLine(true);\noutputter.outputJSONJFObject((foam.core.FObject) user);\noutputter.flush();\n\nfilejournal = new foam.dao.FileJournal(x);\nfilejournal.setMultiLineOutput(true);\nfilejournal.setFile(file);\n\nfirstread = filejournal.getEntry(filejournal.getReader());\nfirstread = firstread.substring(2, firstread.length() - 1);\n\nparser = new foam.lib.json.JSONParser();\nfirstparse = parser.parseString(firstread);\n\noutputter.outputJSONJFObject((foam.core.FObject) firstparse);\nsecondread = filejournal.getEntry(filejournal.getReader());\nsecondread = secondread.substring(2, secondread.length() - 1);\n\ntest(firstread.equals(secondread), \"Reparsing preserves JSON\");\n","output":""})
```

Comment:
Let's format this test as a multiline entry, since we now support that :).  Then it will be easier to review